Makefile

```diff
@@ -229,6 +241,10 @@ llvm_main_file     := driver.md
 llvm_main_filename := $(basename $(notdir $(llvm_main_file)))
 llvm_kompiled      := $(llvm_dir)/$(llvm_main_filename)-kompiled/interpreter
 
+ifeq ($(UNAME_S),Darwin)
+$(KEVM_LIB)/$(llvm_kompiled): $(libcryptopp_out)
+endif
```

Comment:
```suggestion
ifeq ($(UNAME_S),Darwin)
$(KEVM_LIB)/$(haskell_kompiled): $(libcryptopp_out)
$(KEVM_LIB)/$(llvm_kompiled): $(libcryptopp_out)
endif
```

Reply:
I can build without depending on both, but it does no harm to build both at one go.